app/models/menu.rb

```diff
@@ -38,5 +38,26 @@ def self.add_dishes_to_cycle(day_in_cycle, dishes)
     def self.copy_to_temp_menu(day_in_cycle,date)
         true
     end
+    
+    def self.get_dishes_by_id(day_in_cycle)
```

Comment:
Is there a plan for somewhere to use this? It looks like it's not currently in use.

Reply:
We don't plan on using it to the best of my knowledge so I'll remove it.